ProcMod.sc

```diff
@@ -239,7 +242,10 @@ ProcMod {
 		currelfunc = releaseFunc;
 		curccctrl = ccCtrl;
 		isRunning.if({server.sendMsg(\n_free, curgroup)});
-		isReleasing.if({oldgroups.do({arg me; server.sendMsg(\n_free, me)})});
+		isReleasing.if({
+			oldgroups.do({arg me; server.sendMsg(\n_free, me)});
+			oldenvbuses.do({arg me; server.controlBusAllocator.free(me)});
```

Comment:
These shouldn't be freed while it is releasing... This should happen in 'clear'

Reply:
I wasn't super clear on the logic here, but current group is added to `oldgroups` array when the proc *is releasing*, and the only place where `oldgroups` is being used is in `-kill`, to free resources of all instances of the process I assume.

The current envbus is still freed in [`-clear`](https://github.com/supercollider-quarks/JoshMisc/pull/5/files#diff-22f07476de3ee06686252b90b3528164R269). What do you think?

Reply:
So - I just typed up a big response, and while doing so, I think I figured out where I was incorrect and you are correct :) - thanks for your patience on that!